core/src/test/java/org/apache/iceberg/TestRemoveSnapshots.java

```diff
@@ -1143,4 +1145,66 @@ public void testExpireWithDefaultSnapshotAge() {
         Sets.newHashSet(firstSnapshot.manifestListLocation(), secondSnapshot.manifestListLocation()),
         deletedFiles);
   }
+
+  @Test
+  public void testExpireWithDeleteFiles() {
+    Assume.assumeTrue("Delete files only supported in V2 spec", formatVersion == 2);
+    table.updateProperties()
```

Comment:
Do we need to change the age here?

Reply:
Sorry must have slipped in from copy and paste, nice catch, done